website/docs/reference/halving.md

```diff
@@ -0,0 +1,39 @@
+---
+id: halving
+title: Halving
+---
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+The base issuance of CKBytes are rewarded to miners for securing the network, and the reward halves approximately every **4 years** until all the base issuance tokens (**33.6 billion**) are mined out.
+
+## How CKByte-Halving works?
+
+In order to make the halving plan work as expected, a concept of time called **epoch** was introduced.
+
+>An epoch is a period of time for a set of blocks. 
+>
+>In Nervos, the PoW difficulty changes on a new epoch. All the blocks in the same epoch share the same difficulty target. The difficulty adjustment algorithm aims to stabilize the orphan block rate at 2.5% and the epoch duration at 4 hours.
+
+Epochs per halving is `4 * 365 * (24 / 4)` = `8760`, and the Nth halving of CKBytes firstly occurs on epoch: `the_Nth_halving_epoch = 8760 * N `.
+
+So, The CKByte halving event occurs on the specified epoch, e.g. 8760, 17520. 
+
+## When will CKByte be halved?
+The following are the next 10 halving windows of CKBytes, and the corresponding base issuance rewards.
+
+
+|Event          |Date                  |Epoch number      |Epoch reward  |Daily reward  |Total new CKB between events  |
+|---------------|----------------------|------------------|-------------:|--------------:|----------------------------:|
+|Nervos launches|16 November 2016      |0 (genesis epoch) |1,917,808 CKB |11,506,849 CKB |16,800,000,000 CKB           |
+|**First halving**  |**Expected November 2023**|**8,760**             |**958,904 CKB**   |**5,753,424 CKB**  |**8,400,000,000 CKB**            |
+|Second halving |Expected November 2027|17,520            |479,452 CKB   |2,876,712 CKB  |4,200,000,000 CKB            |
+|3rd halving    |Expected November 2031|26,280            |239,726 CKB   |1,438,356 CKB  |2,100,000,000 CKB            |
+|4th halving    |Expected November 2035|35,040            |119,863 CKB   |719,178 CKB    |1,050,000,000 CKB            |
+|5th having     |Expected November 2039|43,800            |59,932 CKB    |359,589 CKB    |525,000,000 CKB              |
+|6th having     |Expected November 2043|52,560            |29,966 CKB    |179,794 CKB    |262,500,000 CKB              |
+|7th having     |Expected November 2047|61,320            |14,983 CKB    |89,897 CKB     |131,250,000 CKB              |
+|8th having     |Expected November 2051|70,080            |7,491 CKB     |44,948 CKB     |65,625,000 CKB               |
+|9th having     |Expected November 2055|78,840            |3,746 CKB     |22,474 CKB     |32,812,500 CKB               |
+|10th having    |Expected November 2059|87,600            |1,873 CKB     |11,237 CKB     |16,406,250 CKB               |
+
```

Comment:
There is one information I would like to add here, that is, how many times halving will the block rewards stop?
The consensus protocol does not specify how many times it will stop, but if the reward of a block is less than 61 ckb, then the reward of the block will stop.
So if we count up to 1800 blocks in an epoch, after the 4th halving, then the base block rewards may not be issued

Reply:
Thanks, I'll add this **important** information to the document.

Reply:
>So if we count up to 1800 blocks in an epoch***
-------------
I found that not every epoch contains 1800 blocks, most of them are less than 1800 blocks, why is that? Is it possible to fix the number of blocks in each epoch to 1800 in the future?

Reply:
> I found that not every epoch contains 1800 blocks, most of them are less than 1800 blocks, why is that? Is it possible to fix the number of blocks in each epoch to 1800 in the future?

Block time is variable, so the number of blocks within an epoch must also be variable.

Reply:
Matt Quinn created a [spreadsheet](https://docs.google.com/spreadsheets/d/1G8eofv1qqs96aEUqEk9vZ8qxeUqSn31KCNoQlwLMyJA/edit?usp=sharing) to show the total supply over time. However, I'm not sure if this includes the rule about not issuing when rewards are less than 61 CKB. @matt-quinn Do you know about the < 61 CKB reward rule?

Reply:
Great job. ![image](https://user-images.githubusercontent.com/87553548/216797456-933e645f-2e86-429a-9708-c2f3227d8449.png)

Nearly 70% of secondary issues have been burned (not issued out), these should not be counted in the circulating supply.

Reply:
> There is one information I would like to add here, that is, how many times halving will the block rewards stop? The consensus protocol does not specify how many times it will stop, but if the reward of a block is less than 61 ckb, then the reward for this block will not be issued. So if we count up to 1800 blocks in an epoch, after the 4th halving, then the base block rewards may not be issued.

fixed: https://github.com/nervosnetwork/docs-new/pull/159/commits/882256f30308f7e6327bf845f7c20019c988c665